Dockerfile

```diff
@@ -460,13 +467,14 @@ RUN mkdir -p /test && \
         --grpc-cpp_out=/test \
         --grpc-csharp_out=/test \
         --grpc-go_out=/test \
-        --grpc-java_out=/test \
+        # --grpc-java_out=/test \
```

Comment:
Is there a reason this is disabled?

Reply:
`grpc-java` is broken due to this: https://gitlab.alpinelinux.org/alpine/aports/-/merge_requests/46676